frontend/packages/console-app/src/index.ts

```diff
@@ -0,0 +1 @@
+import '@console/internal/i18n.js';
```

Comment:
Why do you need this?

Reply:
I followed the i18n doc which said this is needed to generate locale files in the root of the package.

Reply:
thanks I wasn't aware